### PR TITLE
fixed functional tests

### PR DESCRIPTION
**Pull request by leobcgov**
# Fixes or Implements [AB#9960](https://qslvic.visualstudio.com/304a1f8c-dace-4f85-adf3-bf563d5b3a39/_workitems/edit/9960)

* [ ] Enhancement
* [X] Bug
* [ ] Documentation

## Description
- Fixed functions tests.
image

If you are reviewing this PR, please adhere to the guidelines as [documented](https://github.com/bcgov/healthgateway/wiki/prguidance).

## Testing

* [ ] Unit Tests Updated
* [X] Functional Tests Updated
* [ ] Not Required

### Steps to Reproduce

If this is a bug, please provide details on how to reproduce the code.

### UI Changes

NO

### Browsers Tested

* [ ] Chrome
* [ ] Safari
* [ ] Edge
* [ ] Firefox

Provide an explanation for any test(s) not completed.

## Notes/Additional Comments

Any additional notes or comments that may be relevant.  Include any specialized deployment steps here.

### CODE REVIEW

**Comment by leobcgov** [comment]
We need to clean the list of dependents from DEV to make the "Verify Put Dependent Not Found" working.

File: Testing/functional/e2e/cypress/integration/services/webClient/comment.js [hunk 1]
```diff
@@ -35,24 +35,22 @@ describe('WebClient Comment Service', () => {
     }) 
 
     it('Verify Get Comment Authorized', () => {
-        cy.fixture('WebClientService/comments.json').then((expectedResponse) => {
```
Comment (FuriousLlama) [inline Testing/functional/e2e/cypress/integration/services/webClient/comment.js:38]: this is not validating the content of the comments anymore

Comment (leobcgov) [reply Testing/functional/e2e/cypress/integration/services/webClient/comment.js:38]: No I don't think we want to make this test depending on a specific list of comments right? Currently, there is a comment in the dev_db which fails this test.
Unless we want to insert comments in these tests then validate the list.
@FuriousLlama @sslaws @tiagograf-AOT any thoughs?

Comment (leobcgov) [reply Testing/functional/e2e/cypress/integration/services/webClient/comment.js:38]: per our team discussion today, I will add a script to the seed.sql to cleanup comments from db and revert code back to the ogirinal one.

File: Testing/functional/e2e/cypress/integration/timeline/comments.js [hunk 2]
```diff
@@ -10,6 +12,25 @@ function verifyAdd() {
...
+            },
+            headers: {
+                accept: 'application/json'
+            }
+        })
+        .should((response) => { 
+            expect(response.status).to.eq(200)
+            expect(response.body).to.not.be.null
+            cy.log(`response.body: ${JSON.stringify(response.body)}`)
+            expect(JSON.stringify(response.body.resourcePayload)).contains(`"entryTypeCode":"${entryTypeCode}"`);
```
Comment (leobcgov) [inline Testing/functional/e2e/cypress/integration/timeline/comments.js:31]: for info, here is the resourcePayload that we need to validate against. Note that we can't rely on the key "9529523" as it comes from the external service.
`{"resourcePayload":{"9529523":[{"id":"9f33abb2-6bb1-469f-a251-5717612e7e01","userProfileId":"P6FFO433A5WPMVTGM7T4ZVWBKCSVNAYGTWTU3J2LWMGUMERKI72A","text":"test comment goes here!","entryTypeCode":"Med","parentEntryId":"9529523"`

**Review by sslaws — APPROVED** [review]

**Review by tiagograf-AOT — APPROVED** [review]

**Pull request merged**